toffy/qc_comp.py

```diff
@@ -264,14 +266,11 @@ def sort_bin_file_fovs(fovs, suffix_ignore=None):
     )
 
 
-def compute_qc_metrics(bin_file_path, extracted_imgs_path, fov_name,
+def compute_qc_metrics(extracted_imgs_path, fov_name,
```

Comment:
Note that the [generate_qc](https://github.com/angelolab/toffy/blob/d088f740952683fcf8c84367cc322adcd0422c92/toffy/watcher_callbacks.py#L171) FOV callback actually calls `compute_qc_metric_direct` itself, not this function. I think this was leftover from when the function above used the bin files for calculation, so Adam made a "direct" function to skip that re-extraction while running watcher. We should probably adjust the dosctrings of both functions (and possibly the helper function name) to make clear what's going on.